cms/djangoapps/models/settings/course_metadata.py

```diff
@@ -116,12 +126,26 @@ def filtered_list(cls):
         if not switches.is_enabled(u'instructor_paced_only'):
             filtered_list.append('certificate_available_date')
 
-        # Do not show field for configuring Adaptive Learning if feature is disabled.
-        if not AdaptiveLearningEnabledFlag.is_enabled():
-            filtered_list.append('adaptive_learning_configuration')
+        return filtered_list
+
+    @classmethod
+    def course_filtered_list(cls, descriptor):
```

Comment:
I've called it `descriptor` because that's how it's called in other methods (e.g. `fetch()`). Of course `course_descriptor` sounds better, so I'm going to change it :)